app/controllers/spam_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class SpamController < ApplicationController
+  before_action :session_required, only: %i[redirect_to_first_potential_spammer]
+  before_action :admin_session_required, only: %i[redirect_to_first_potential_spammer]
+
+  def redirect_to_first_potential_spammer
+    sql = <<-SQL.squish
+             SELECT id FROM oh.potential_spammers LIMIT 1;
+    SQL
```

Comment:
This can be simplified to a single line:
```
sql = 'Select id FROM...'
```

Reply:
Fixed.

Reply:
Also fixed this in the test.